app/views/handlers/content.py

```diff
@@ -47,3 +44,9 @@ def _get_content_title(self, transformed_block):
         content = transformed_block.get("content")
         if content:
             return self._get_safe_page_title(content["title"])
+
+    def _is_first_block_in_individual_response(self, location, schema):
```

Comment:
Might be better suited as `is_block_first_block_in_individual_response`. Also you only need to pass the block_id not the location object.

Reply:
Ok, done. Renamed and passed only block_id now👌.